main.js

```diff
@@ -21,4 +21,142 @@ closeIcon.addEventListener('click', disapear);
+
+    technologies: ['html', 'css', 'javaScript', 'github', 'ruby', 'Bootstraps'],
+    links: [
+      'https://jord4n-ac.github.io/my-portfolio/#section-2',
+      'https://github.com/Jord4N-AC/my-portfolio',
+    ],
+    icons: [
+      'images/icons/exit-icon-gray.svg',
+      'images/cardsWorksSection/see_live_btn.svg',
+      'images/cardsWorksSection/github_btn.svg',
+    ],
+  },
+];
+
+// Form Validation
+
+function showModal(ind) {
+  const section = document.querySelectorAll('.card');
+
+  const modalHtml = `
+      <div class="modal-container">
+          <div class="header-container">
+              <div>
+                  <h3 class="proj-header">${popupContent[ind].titleProject}</h3>
+                  <img class="x-icon" src="${popupContent[ind].icons[0]}" alt="close window icon">
+              </div>
+              <ul class="features">
+                  <li class="f f1 upper-case">${popupContent[ind].features[0]}</li>
+                  <li class="dot">.</li>
+                  <li class="f f2">${popupContent[ind].features[1]}</li>
+                  <li class="dot">.</li>
+                  <li class="f f3">${popupContent[ind].features[2]}</li>
+              </ul>
+          </div>
+          <div class="img-container">
+              <img class="img-proj" src="${popupContent[ind].imgProject}" alt="project1 image modal">
+          </div>
+          <div class="content-container">
+              <p class="p p-proj">
+                  ${popupContent[ind].contentProject}
+              </p>
+              <div class="tech-btns-container">
+                  <ul class="tags">
+                      <li>${popupContent[ind].technologies[0]}</li>
+                      <li>${popupContent[ind].technologies[1]}</li>
+                      <li>${popupContent[ind].technologies[2]}</li>
+                      <li class="hide">${popupContent[ind].technologies[3]}</li>
+                      <li class="hide">${popupContent[ind].technologies[4]}</li>
+                      <li class="hide">${popupContent[ind].technologies[5]}</li>
+                  </ul>
+                  <div class="btns-container">
+                      <a class="btn sp-button" href="${popupContent[ind].links[0]}">
+                          See live
+                          <img src="${popupContent[ind].icons[1]}" alt="See live icon">
+                      </a>
+                      <a class="btn sp-button" href="${popupContent[ind].links[1]}">
+                          See Source
+                          <img src="${popupContent[ind].icons[2]}" alt="Github icon">
+                      </a>
+                  </div>
+              </div>
+          </div>
+      </div>
+  `;
+
+  section[ind].insertAdjacentHTML('afterbegin', modalHtml);
+
+  const closeModalIcon = document.querySelector('.x-icon');
```

Comment:
- [ ] [OPTIONAL]: You should consider adding a listener to close the modal whenever you click outside it. You can read more about it in [this thread](https://stackoverflow.com/questions/37573608/how-to-make-modal-close-on-click-outside).